recipes/libmatio/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./autogen.sh
+./configure --quiet --enable-shared --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+make install
```

Comment:
Have only proposed. Not actually removed because I'm not sure to what extent we may already be dependent on it. :/